elide-contrib/elide-dynamic-config-helpers/src/main/resources/elideTableSchema.json

```diff
@@ -247,7 +243,7 @@
         "name",
         "type",
         "definition",
-        "grains"
+        "grain"
```

Comment:
Not a typo.
Instead of a set of TimeDimensionGrain, it will have only 1 TimeDimensionGrain as per https://github.com/yahoo/elide/issues/1474
Expected Format is:
           grain:
            {
             type :  DATE
             sql :  '''
             PARSEDATETIME(FORMATDATETIME(${column}, 'yyyy-MM-dd'), 'yyyy-MM-dd')
             '''
            }

Reply:
Here its correct, fixed it few lines above to rename grains->grain.